docs/examples/frameworks/mxnet/mxnet-various-readers.ipynb

```diff
@@ -96,31 +96,26 @@
    "metadata": {},
```

Comment:
<p>TFRecord reader pipeline (the camel-cased name doesn't make sense now)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/2761/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Done</p>